packages/ui/src/components/EasySetup/ManualExtrinsic.tsx

```diff
@@ -175,21 +224,19 @@ const ManualExtrinsic = ({
     let paramFields: ParamField[] = []
     const metaArgs = api.tx[palletRpc][callable].meta.args
 
-    console.log('metaArgs', metaArgs)
+    // console.log('metaArgs', metaArgs)
     if (metaArgs && metaArgs.length > 0) {
       paramFields = metaArgs.map((arg) => {
-        console.log('getTypeDef', getTypeDef(arg.type.toString()))
-        const instance = api.registry.createType(arg.type as unknown as 'u32')
-        console.log('instance', instance)
-        const raw = getTypeDef(instance.toRawType())
-        console.log('raw', raw)
-
-        arg.typeName.isSome &&
-          console.log('typeName.unwrap().toString()', arg.typeName.unwrap().toString())
+        // console.log('getTypeDef', getTypeDef(arg.type.toString()))
+        // const instance = api.registry.createType(arg.type as unknown as 'u32')
+        // console.log('instance', instance)
+        // const raw = getTypeDef(instance.toRawType())
+        // console.log('raw', raw)
```

Comment:
can we maybe move it to the log method and comment it?

Reply:
After offline discussion we're keeping this around for a bit.